translations/all_map_locations.tex/he.po

```diff
@@ -85,6 +87,30 @@ msgid ""
 "\\endgroup\n"
 "\n"
 msgstr ""
+"\\begingroup\n"
+"  \\renewcommand{\\arraystretch}{1.5}\n"
+"  \\newcolumntype{Y}{>{\\centering\\arraybackslash}X}\n"
+"  \\begin{tabularx}{\\linewidth}{Y m{0.8\\linewidth}}\n"
+"    \\small\n"
+"    \\mbox{\\textbf{\\textenglish{I-VII}}} & \\texthebrew{רמת הקושי של השדה, מציינת את הרמה של היחידות הנטרליות הנמצאות בשדה הזה.}\\smallskip \\\\\n"
+"    \\svg[15]{treasure} & \\texthebrew{\\parbox{0.7\\hsize}{\n"
+"      הטל \\pagetarget{Treasure Die}{קובית אוצרות}\\index{Treasure Die}\\index{Dice} וקבל את הבונוס המצויין עליה (\\svg{resource_die}, \\svg{2_treasure_die}, \\svg{experience}, או \\svg{artifact}).\n"
+"    }}\n"
+"    \\begin{tikzpicture}[overlay]\n"
+"      \\node at (1, 0.25) {\\includegraphics[width=0.2\\linewidth]{\\images/treasure_die.png}};\n"
+"    \\end{tikzpicture} \\\\\n"
+"    \\svg[15]{resource_die} & \\texthebrew{הטל \\pagelink{Resources}{קוביית משאבים} וקבל את המשאב המצויין עליה.} \\\\\n"
+"    \\svg[15]{2_treasure_die} & \\texthebrew{הטל 2 קוביות משאבים ובחר את הבונוס המצויין על אחת מהן.} \\\\\n"
+"    \\svg[15]{experience} & \\texthebrew{קבל נקודת ניסיון אחת.} \\\\\n"
```

Comment:
```suggestion
"    \\texthebrew{קבל נקודת ניסיון אחת.} & \\svg[15]{experience}\\\\\n"
```
Firstly, I think you'll have to swap those.

Reply:
I tried
it looks terrible

<img width="449" alt="image" src="https://github.com/user-attachments/assets/6a2120fb-141e-4e39-b165-f20889c79f7b">

Reply:
Hang on, it's an easy fix.